docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -23,41 +23,49 @@ Synopsis
 
 .. code-block:: cpp
 
+    struct Kokkos::ChunkSize {
+        ChunkSize(int value_);
+    };
+
     template<class ... Args>
-    class Kokkos::RangePolicy {
-        typedef RangePolicy execution_policy;
-        typedef typename traits::index_type member_type ;
-        typedef typename traits::index_type index_type;
+    struct Kokkos::RangePolicy {
+        using execution_policy = RangePolicy;
+        using member_type = PolicyTraits<Args...>::index_type;
 
-        //Inherited from PolicyTraits<Args...>
+        // Inherited from PolicyTraits<Args...>
         using execution_space   = PolicyTraits<Args...>::execution_space;
         using schedule_type     = PolicyTraits<Args...>::schedule_type;
         using work_tag          = PolicyTraits<Args...>::work_tag;
         using index_type        = PolicyTraits<Args...>::index_type;
         using iteration_pattern = PolicyTraits<Args...>::iteration_pattern;
         using launch_bounds     = PolicyTraits<Args...>::launch_bounds;
 
-        //Constructors
+        // Constructors
         RangePolicy(const RangePolicy&) = default;
         RangePolicy(RangePolicy&&) = default;
 
-        inline RangePolicy();
+        RangePolicy();
+
+        RangePolicy( member_type work_begin
```

Comment:
I'd say `index_type`
I think using member type was an attempt to have genericity between exec policy but it clearly does not work with MDRangePolicy.

Reply:
Changed to `index_type`.